### PR TITLE
CIV-14644 application status

### PR DESCRIPTION
**Pull request by GorilaNaranja**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-14644

### Change description ###
Application status - Summary page - GA for LiPs - Post Court decision

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/common/models/generalApplication/applicationSummary.ts [hunk 1]
```diff
@@ -3,11 +3,26 @@ export enum ApplicationState {
   AWAITING_APPLICATION_PAYMENT = 'AWAITING_APPLICATION_PAYMENT',
   APPLICATION_SUBMITTED_AWAITING_JUDICIAL_DECISION = 'APPLICATION_SUBMITTED_AWAITING_JUDICIAL_DECISION',
   APPLICATION_ADD_PAYMENT = 'APPLICATION_ADD_PAYMENT',
+  PENDING_APPLICATION_ISSUED = 'PENDING_APPLICATION_ISSUED',
+  ADDITIONAL_RESPONSE_TIME_EXPIRED = 'ADDITIONAL_RESPONSE_TIME_EXPIRED',
+  ADDITIONAL_RESPONSE_TIME_PROVIDED = 'ADDITIONAL_RESPONSE_TIME_PROVIDED',
+  AWAITING_DIRECTIONS_ORDER_DOCS = 'AWAITING_DIRECTIONS_ORDER_DOCS',
+  ORDER_MADE = 'ORDER_MADE',
```
Comment (manish14836) [inline src/main/common/models/generalApplication/applicationSummary.ts:10]: Suggestion:- Sort these

File: src/main/modules/i18n/locales/cy.json [hunk 2]
```diff
@@ -2820,28 +2820,30 @@
...
+          "AWAITING_DIRECTIONS_ORDER_DOCS": "Awaiting directions order document",
+          "ORDER_MADE": "Order made",
+          "LISTING_FOR_A_HEARING": "Listing for a hearing",
           "HEARING_SCHEDULED": "Hearing Scheduled",
           "APPLICATION_PAYMENT_FAILED": "Application Payment Failed",
           "AWAITING_APPLICATION_PAYMENT": "Awaiting Application Payment",
-          "AWAITING_WRITTEN_REPRESENTATIONS": "Awaiting Written Representations",
-          "AWAITING_ADDITIONAL_INFORMATION": "Additional Information Required",
-          "APPLICATION_DISMISSED": "Application Dismissed",
-          "APPLICATION_CLOSED": "Application Closed",
```
Comment (manish14836) [inline src/main/modules/i18n/locales/cy.json:2843]: Sort these

File: src/main/common/models/generalApplication/applicationSummary.ts (hunk 1 above)
Comment (GorilaNaranja) [reply src/main/common/models/generalApplication/applicationSummary.ts]: done

File: src/main/modules/i18n/locales/cy.json (hunk 2 above)
Comment (GorilaNaranja) [reply src/main/modules/i18n/locales/cy.json:2843]: done

**Review by manish14836 — APPROVED** [review]

**Review by paul-pearson — APPROVED** [review]

**Pull request merged**